src/test/resources/config.yml

```diff
@@ -97,5 +97,5 @@ config:
     hosts_resolver_mode: "ip-only"
     transport_userrname_attribute: null
     on_behalf_of:
-      signing_key: "signing key"
-      encryption_key: "encryption key"
+      signing_key: "VGhpcyBpcyB0aGUgand0IHNpZ25pbmcga2V5IGZvciBhbiBvbiBiZWhhbGYgb2YgdG9rZW4gYXV0aGVudGljYXRpb24gYmFja2VuZCBmb3IgdGVzdGluZyBvZiBleHRlbnNpb25z"
```

Comment:
For this and the files below, are these settings required in all of those test config files?

Reply:
Yes, I believe these files we will need them to run the tests. Remember those `signing key` caused some Base64 decoding issue when there was a space in between.

Reply:
I'm confused why the change is necessary in 5 of the test config files. Is it possible to have a config file specifically for the tests added in this PR and configure the settings in that file?